content/updates/automation/4-Enhanced-rule-management.md

```diff
@@ -0,0 +1,32 @@
+# Enhanced rule management
+
+## Quickly duplicate your rules
+::: meta-data type="New" ee-only="true" link-to-doc="../articles/manage-your-rules.html#duplicate-a-rule"
+:::
+
+Sometimes, you may have very similar rules with only one condition that is different. In order to avoid re-writing the whole rule, you are now able to duplicate it in two clicks! What a time saver, isn't it? :wink:
```

Comment:
```suggestion
Sometimes you can have very similar rules with just one different condition. In order to avoid rewriting the whole rule, you can now duplicate it in two clicks! What a time-saver, isn't it?
```